sectionctl.go

```diff
@@ -102,8 +114,9 @@ func main() {
 		kong.UsageOnError(),
 		kong.ConfigureHelp(kong.HelpOptions{Tree: true}),
 	)
-	bootstrap(cli, ctx)
-	err := ctx.Run()
+	contxt := bootstrap(cli, ctx)
```

Comment:
Don't use `context` as variable name.

Reply:
Call this `ctx` and the other one `kongCtx` to make a distinction.

Reply:
I've renamed the kong `ctx` to `cmd`, since that is more accurate to the information that it holds.